packages/occ-worker/src/actions.ts

```diff
@@ -63,11 +63,14 @@ function buildModel(
   return outputModel;
 }
 
-function loadFile(payload: { content: IJCadContent }): IDict | null {
+function loadFile(
+  payload: { content: IJCadContent },
+  load_on_failure = false
+): IDict | null {
   const { content } = payload;
   const outputModel = buildModel(content);
 
-  const parser = new OccParser(outputModel);
+  const parser = new OccParser(outputModel, load_on_failure);
```

Comment:
I feel more natural to use the `loadOnFailure` parameter as the `parser. execute` parameter